lib/utils/encryptor.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+const crypto = require('crypto');
+const buffer1 = require('buffer');
+
+const Encryptor = {
+  /**
+   * Calculate a hash of the concatenated buffers with the given algorithm.
+   * @param {string} algorithm - The hash algorithm.
+   * @param {Array.<Buffer>} buffers - The buffers to concat and hash
+   * @returns {Buffer} The hash
+   */
+  hash(algorithm) {
```

Comment:
If the function signature were hash(algorithm, ...buffers), we can eliminate the next 4 lines